contracts/test/gateway/MockUtilityToken.sol

```diff
@@ -52,7 +56,9 @@ contract MockUtilityToken is UtilityToken {
     )
         public
         UtilityToken(_token, _symbol, _name, _decimals, _organization)
-    {}
+    {
+        balances[msg.sender] = INITIAL_BALANCE;
```

Comment:
We should also add to make it consistent.
`totalTokenSupply = totalTokenSupply.add(_amount);`